src/trove/chart.js.rkt

```diff
@@ -616,38 +1563,139 @@ a-series = from-list.labeled-histogram(
     we let users increase sample sizes, allowing the function to be rendered
     more accurately.
   }
+  
+  @repl-examples[
+   `(@{NUM_E = ~2.71828
+f-series = from-list.function-plot(lam(x): 1 - num-expt(NUM_E, 0 - x) end)
+render-chart(f-series).display()} ,(in-image "function-plot-example"))
+  ]
+
+  @method-doc["DataSeries" "function-plot-series" "color"]
+  @repl-examples[
+   `(@{include color
+     render-chart(f-series.color(orange)).display()} ,(in-image "function-plot-color-example"))
+  ]
+
+  @method-doc["DataSeries" "function-plot-series" "legend"]
+  @repl-examples[
+   `(@{render-chart(f-series.legend("My Legend")).display()} ,(in-image "function-plot-legend-example"))
+  ]
+
+  @;################################
+  @subsection{Line Plot Series}
+
+  @constructor-doc["DataSeries" "line-plot-series" opaque DataSeries]{
+    A line plot series
+  }
+  @repl-examples[
+   `(@{lineplot-series = from-list.line-plot(
+  [list: 0,  1, 2,  3, 6, 7,  10, 13, 16, 20],
+  [list: 18, 2, 28, 9, 7, 29, 25, 26, 29, 24])
+render-chart(lineplot-series).display()} ,(in-image "line-plot-example"))
+  ]
+
+  @method-doc["DataSeries" "line-plot-series" "color"]
+  @repl-examples[
+   `(@{include color
+render-chart(lineplot-series.color(orange)).display()} ,(in-image "line-plot-color-example"))
+  ]
+
+  @method-doc["DataSeries" "line-plot-series" "legend"]
+  @repl-examples[
+   `(@{render-chart(lineplot-series.legend("My Legend")).display()} ,(in-image "line-plot-legend-example"))
+  ]
+
+  @method-doc["DataSeries" "line-plot-series" "labels"]
+  @repl-examples[
+   `(@{render-chart(lineplot-series.labels(
+                    [list: "a", "b", "c", "d", "e", "f", "g", "h", "i", "j"])
+                   ).display()} ,(in-image "line-plot-labels-example"))
+  ]
+
+  @method-doc["DataSeries" "line-plot-series" "image-labels"]
+  @repl-examples[
+  `(@{include image
+      render-chart(lineplot-series.image-labels(
+                    [list: circle(30, "outline", "red"), 
+                           circle(30, "solid", "red"),
+                           circle(30, "outline", "blue"), 
+                           circle(30, "solid", "blue"),
+                           circle(30, "outline", "green"), 
+                           circle(30, "solid", "green"),
+                           circle(30, "outline", "purple"), 
+                           circle(30, "solid", "purple"),
+                           circle(30, "outline", "black"),
+                           circle(30, "solid", "black")])
```

Comment:
There's a typo here with an extra close parenthesis

Reply:
Sorry I don't see it. Could you tell me which line it's on?